src/Tasks/Microsoft.NET.Build.Tasks/ResolvePublishAssemblies.cs

```diff
@@ -32,7 +33,9 @@ public class ResolvePublishAssemblies : TaskBase
 
         public bool PreserveCacheLayout { get; set; }
 
-        public string FilterProjectAssetsFile { get; set; }
+        public bool DoNotTrackPackageAsResolved { get; set; }
+
+        public string FilterProjFiles { get; set; }
```

Comment:
Avoid abbreviations. FilterProjectFiles?

Also, avoid splitting on semicolons manually, it is really hard to follow. Use array type for non-scalar msbuild input and msbuild will handle splitting for you.